js/initcontext.go

```diff
@@ -239,7 +240,8 @@ func (i *InitContext) Open(ctx context.Context, filename string, args ...string)
 	}
 
 	if len(args) > 0 && args[0] == "b" {
-		return i.runtime.ToValue(data), nil
+		ab := i.runtime.NewArrayBuffer(data)
```

Comment:
here @imiric , there is no point in having `ab` ;) 

This is done in another 5 places probably

Reply:
Hhmm how would I get the pointer to it otherwise? Parenthesis wouldn't work in this case.

Though I did also spot `msg := message.Export()` which is not needed.

Reply:
I also forget that you can't directly get the pointer from the function call :facepalm: 

On the other hand it seems like ... you don't need to? I mean do you really need to give a pointer?

Reply:
I err'd on the assumption that it's better to pass around pointers to this data than incur copying penalties, but considering the [internal implementation is a pointer](https://github.com/dop251/goja/blob/1b11a6af686d7af5badfdd6588364537db72df09/typedarrays.go#L37) I guess it would be safe to return the struct itself.

Do you think this is worthwhile changing?

Reply:
as my original comment suggests, I don't think it is a requirement.

In general, this means that it will now need to dereference 2 pointers in order to get to the actual data which is objectively worse, but not enough to matter in the grand scheme of things